content/guides/mobile/ios/quick-start/2-install-ios-sdk.md

```diff
@@ -0,0 +1,117 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Install the iOS SDK
+
+With an iOS application in place, you will need to import the required **Box
+iOS SDK** dependencies into your project using one of the available package
+manager options.
+
+## Select a package manager to use
+
+<Grid columns='3'>
+  <Choose option='ios.pm_type' value='carthage' color='blue'>
+    # Carthage
+
+    Carthage is a decentralized dependency manager for Swift
+    and Objective-C Cocoa projects. It is open-source and built
+    with Swift.
+  </Choose>
+
+  <Choose option='ios.pm_type' value='cocoapods' color='none'>
+    # CocoaPods
+
+    CocoaPods is a centralized dependency manager for Swift and
+    Objective-C Cocoa projects. It is open-source and was built
+    with Ruby.
+  </Choose>
+
+  <Choose option='ios.pm_type' value='swift' color='blue'>
+    # Swift Package Manager
+
+    Included with Swift since version 3.0, the Swift Package
+    Manager is a tool for managing the distribution of source
+    code, aimed at making it easy to share your code and reuse
+    others’ code.
+  </Choose>
+</Grid>
+
+<Choice option='ios.pm_type' value='carthage' color='blue'>
+  # Install the iOS SDK using Carthage
+
+  1. From a terminal window, install Carthage: `brew install carthage`.
+    [See here](https://github.com/Carthage/Carthage#installing-carthage)
```

Comment:
Generally "here" is considered bad practice in docs. A better option might be "Head over to the Carthage documentation for other installation methods"